src/main/java/sirius/biz/tycho/DashboardUserAgentsChart.java

```diff
@@ -0,0 +1,91 @@
+import sirius.kernel.commons.Strings;
+import sirius.kernel.di.std.Register;
+import sirius.web.security.Permission;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.function.Consumer;
+
+/**
+ * Implements a time series chart for the dashboard.
+ *
+ * The chart visualizes the browser which have been used in order to access the dashboard by extracting the required data from the user agents dataset.
+ */
+@Register
+@Permission(TenantUserManager.PERMISSION_SYSTEM_TENANT_MEMBER)
+public class DashboardUserAgentsChart extends TimeSeriesChartFactory<Object> {
+
+    private static final String SQL_QUERY = Strings.apply(// language=SQL
+                                                          """
+                                                                  SELECT %s, %s, %s, %s, %s, YEAR(eventDate) as year, MONTH(eventDate) AS month [:daily , DAY(eventDate) AS day]
+                                                                  FROM pageimpressionevent
+                                                                  WHERE aggregationUri = '/system/dashboard' AND eventDate >= ${start} AND eventDate <= ${end}
+                                                                  GROUP BY [:daily DAY(eventDate), ] MONTH(eventDate), YEAR(eventDate)                                                     
+                                                                  """,
+                                                          UserAgentsTimeSeriesComputer.COUNT_FIREFOX,
+                                                          UserAgentsTimeSeriesComputer.COUNT_CHROME,
+                                                          UserAgentsTimeSeriesComputer.COUNT_INTERNET_EXPLORER,
+                                                          UserAgentsTimeSeriesComputer.COUNT_SAFARI,
+                                                          UserAgentsTimeSeriesComputer.COUNT_EDGE);
+
+    @Nullable
+    @Override
+    protected Class<? extends ChartObjectResolver<Object>> getResolver() {
+        return null;
+    }
+
+    @Override
+    public String getCategory() {
+        return StandardCategories.MISC;
+    }
+
+    @Override
+    protected void collectReferencedCharts(Consumer<Class<? extends ChartFactory<Object>>> referenceChartConsumer) {
+        // intentionally empty -> there are no referenced charts
+    }
+
+    @Override
+    protected void computers(boolean hasComparisonPeriod,
+                             boolean isComparisonPeriod,
+                             Callback<TimeSeriesComputer<Object>> executor) throws Exception {
+        if (isComparisonPeriod) {
+            return;
+        }
+
+        executor.invoke(new UserAgentsTimeSeriesComputer<>(ignored -> SQL_QUERY));
+    }
+
+    @Nonnull
+    @Override
+    public String getName() {
+        return "GlobalDashboardUserAgentsChart";
+    }
+
+    @Override
+    public int getPriority() {
+        return 0;
```

Comment:
```suggestion
        return 900;
```

I'd rather rank this chart quite low as there might be more interesting charts available...